reference/7.3/Microsoft.PowerShell.Core/About/about_Logging_Non-Windows.md

```diff
@@ -230,45 +228,38 @@ The following code is an example configuration:
 }
 ```
 
-The properties for configuring PowerShell logging are listed in the following
-table. Values marked with an asterisk, such as `Operational*`, indicate the
-default value when no value is provided in the file.
-
-|Property   |Values        |Description                                  |
-|-----------|--------------|---------------------------------------------|
-|`LogIdentity`|(string name) |The name to use when logging. By default,  |
-|           |powershell*   |powershell is the identity. This value can be|
-|           |              |used to tell the difference between two      |
-|           |              |instances of a PowerShell installation, such |
-|           |              |as a release and beta version. This value is |
-|           |              |also used to redirect log output to a        |
-|           |              |separate file on Linux. See the discussion of|
-|           |              |**rsyslog** above.                           |
-|`LogChannels`|Operational*  |The channels to enable. Separate the values|
-|           |Analytic      |with a comma when specifying more than one.  |
-|`LogLevel`   |Always        |Specify a single value. The value enables  |
-|           |Critical      |itself and all values above it in the        |
-|           |Error         |list to the left.                            |
-|           |Warning       |                                             |
-|           |Informational*|                                             |
-|           |Verbose       |                                             |
-|           |Debug         |                                             |
-|`LogKeywords`|Runspace      |Keywords provide the ability to limit logging|
-|           |Pipeline      |to specific components within PowerShell. By |
-|           |Protocol      |default, all keywords are enabled and change |
-|           |Transport     |this value is only useful for very           |
-|           |Host          |specialized troubleshooting.                |
-|           |Cmdlets       |                                             |
-|           |Serializer    |                                             |
-|           |Session       |                                             |
-|           |ManagedPlugin |                                             |
+The following is a list of properties for configuring PowerShell logging.
+Values marked with an asterisk, such as `Operational*`, indicate the default
+value when no value is provided in the file.
+
+- `LogIdentity`
```

Comment:
As properties, I think these should be bolded instead of backticked - this will also help look less confusing when the values are backticked instead of bare strings.
```suggestion
- **LogIdentity**
```